### PR TITLE
Begin spring 2018 course page

### PR DESCRIPTION
**Pull request by abizer**
I changed the format of materials.yaml and calendar.yaml a bit

### CODE REVIEW

File: _includes/calendar.html [hunk 1]
```diff
@@ -3,39 +3,50 @@ <h2 class="page-header" id="calendar">Calendar</h2>
...
+        {% for item in site.data.materials %}
+          {% if item.id == entry.id %}
+      <td>
+              {% if item.slides %}
+          <a href="{{ item.slides }}">
+              {% endif %}
+              {{ item.name | default: - }}
+              {% if item.slides %}
+          </a>
+              {% endif %}
```
Comment (abizer) [inline _includes/calendar.html]: emacs is garbage sometimes with indentation

File: _data/calendar.yaml [hunk 2]
```diff
@@ -4,78 +4,20 @@
...
-  lab: 0
+  id: 0
 
 - date: 2017-09-07
+  week: 1
+  id: b1
+  track: Basic
 
-- date: 2017-09-12
+- date: 2017-09-09
```
Comment (Baisang) [inline _data/calendar.yaml]: Dates here is wrong

File: index.md [hunk 3]
```diff
@@ -5,19 +5,43 @@
...
 
-An important skill for system administrators is the **willingness to learn quickly** about new and unfamiliar technologies, so while we expect many students will have a CS background, **the only real prerequisite is a desire to learn**. We assume **no prior experience** and will get started and move quickly - there's a lot of material to cover. Another important skill to learn is patience and perseverence. Don't be discouraged if things don't work: the internet is held together with the programmatic equivalent of duct tape, and the job of a sysadmin is best described as applying new duct tape when the old stuff wears out, which happens a lot.
+An important skill for system administrators is the **willingness to learn
+quickly** about new and unfamiliar technologies, so while we expect many
+students will have a CS background, **the only real prerequisite is a desire
+to learn**. We know potential students will have varied backgrounds in terms
+of CS and sysadmin experience, therefore, we will be teaching this course in
+two tracks: a beginner's track that will focus more on introducing new users
+to GNU/Linux, and an advanced track that will cater to students with more
```
Comment (Baisang) [inline index.md]: or as I've been referring to it recently, GNU+Linux

File: _data/calendar.yaml [hunk 4]
```diff
@@ -4,78 +4,20 @@
...
-  lecture: 9
-  lab: 9
-
-- date: 2017-11-14
-  week: 10
-  lecture: 10
-  lab: 10
-  primary: True
-  signin: https://goo.gl/forms/HMtkThja0XriLeeX2
+  note: Cancelled, MLK building closed.
```
Comment (jvperrin) [inline _data/calendar.yaml]: Why is this here? Is the building closed in January or something?

File: _data/materials.yaml [hunk 5]
```diff
@@ -3,44 +3,16 @@
 - id: 0
   name: Info Session
   slides: https://docs.google.com/presentation/d/1T3x-K1YJEyDw8fUEX9nsrEkJ4BXwW7d-yffKF0Mb5VE/edit?usp=sharing
```
Comment (jvperrin) [inline _data/materials.yaml]: I feel like these slides links should be removed, because I imagine we'd want to copy last semester's slides and labs without modifying the originals, so the link will change. I guess these can be kept here for now, but I think just having dummy links for now might make it clearer that the slides don't actually exist yet.

File: _includes/calendar.html [hunk 6]
```diff
@@ -3,39 +3,50 @@ <h2 class="page-header" id="calendar">Calendar</h2>
   <table class="table table-striped">
     <thead>
       <tr>
-	<th>Week</th>
-	<th>Date</th>
-	<th>Lecture (Tu/Th)</th>
-	<th>Lab</th>
+	    <th>Week</th>
+	    <th>Date</th>
+        <th>Track</th>
```
Comment (jvperrin) [inline _includes/calendar.html]: Indentation

File: _includes/calendar.html [hunk 7]
```diff
@@ -3,39 +3,50 @@ <h2 class="page-header" id="calendar">Calendar</h2>
...
-		  {% endif %}
-		{% endfor %}
-	      </a>
-	    {% elsif entry.lecturenote %}
-	      {{ entry.lecturenote }}
-	    {% else %} -
-	    {% endif %}
+	  <td>{{ entry.week  | default: - }}</td>
+	  <td>{{ entry.date  | date: "%a. %m/%d" }}</td>
+      <td>{{ entry.track | default: - }}</td>
```
Comment (jvperrin) [inline _includes/calendar.html]: Indentation here too

File: _includes/calendar.html (hunk 1 above)
Comment (jvperrin) [reply _includes/calendar.html]: `!sux emacs indentation`

File: _layouts/home.html [hunk 8]
```diff
@@ -32,15 +32,14 @@
...
 		  <a href="mailto:decal@ocf.berkeley.edu">Contact</a>
 		</li>
-		<!-- Commented out because the interest form is now closed -->
-		<!-- <li>
-	<a href="https://goo.gl/forms/JUQyRdYg8ZgUyXSH3">Interest Form</a>
-	</li> -->
+	    <li>
+	<a href="https://goo.gl/forms/70uze4r9OhWGRuh42">Interest Form</a>
+	</li>
 	<li>
```
Comment (jvperrin) [inline _layouts/home.html]: lol why was the indentation even like this before? *sigh*

File: _layouts/home.html [hunk 9]
```diff
@@ -32,15 +32,14 @@
...
 	  <a href="https://www.ocf.berkeley.edu/">OCF</a>
 	</li>
 	<li>
-	  <a href="http://xcf.berkeley.edu">XCF</a>
+	  <a href="https://xcf.berkeley.edu">XCF</a>
```
Comment (jvperrin) [inline _layouts/home.html]: Nice

File: index.md [hunk 10]
```diff
@@ -5,19 +5,43 @@
...
+* have a good sense about maintaining system security
 * understand system administration essentials
-* get a practical taste for what's new in the field
+* get a practical taste of what sysadmins do in industry.
 
-Lectures are scheduled for **8-9pm Tuesdays and Thursdays** in the OCF Lab, located at **171 MLK** (MLK basement down the hall from the student store). **Attendance is mandatory with two allowed unexcused abscences** (other conflicts such as exams can be excused).
+Lectures are scheduled for **Tuesdays and Thursdays 8-9pm** the OCF Lab,
+located at **171 MLK** (MLK basement down the hall from the student store).
+**Attendance is mandatory, with two allowed unexcused absences.**
+(other conflicts such as exams are excused).
```
Comment (jvperrin) [inline index.md]: I'd say something like `some other conflicts, such as exams, are excused`, since some conflicts would be excusable, and some would not.

File: index.md (hunk 3 above)
Comment (jvperrin) [reply index.md]: lol

File: labs/10.md [hunk 11]
```diff
@@ -1,192 +0,0 @@
----
```
Comment (jvperrin) [inline labs/10.md]: Why are the labs removed in this pull request instead of being moved in a426dfb9fd4ebac1f3993eca0bc274b2c3bdb141 into the archive?

File: _data/calendar.yaml (hunk 2 above)
Comment (abizer) [reply _data/calendar.yaml]: these are all placeholder dates to test the new config file layout

File: _data/calendar.yaml (hunk 4 above)
Comment (abizer) [reply _data/calendar.yaml]: placeholder stuff to test the new config file layout

Comment (jvperrin) [reply _data/calendar.yaml]: Ok, sounds fine then

File: labs/10.md (hunk 11 above)
Comment (abizer) [reply labs/10.md]: The rendered HTML of the labs is in the archives, I just removed the markdown files. Unless you mean something else?

Comment (jvperrin) [reply labs/10.md]: Oh, I just expected the whole contents minus the `archives` directory to be archived in that commit, so the labs and stuff would have been moved too (and removed from here I suppose), but this works fine too.

**Comment by abizer** [comment]
any further thoughts or things we should add? Did I get everything right as far as everyone can tell?
